controllers/errorControllers.js

```diff
@@ -1,4 +1,8 @@
 exports.handle500Errors = (error, req, res, next) => {
     console.log(error)
     res.status(500).send({msg: "Server Error"})
+}
+
+exports.handle404Errors = (error, req, res, next) => {
```

Comment:
There will be other 404 errors other than non existent endpoints later on, so I would consider renaming this handler.